src/models/user/postgres.js

```diff
@@ -97,7 +99,7 @@ const UserRepo = (postgres) => {
 
   const setHours = async (start_hour, end_hour, id) => {
     const values = [start_hour, end_hour, id];
-    // console.log(values); This line is to test that the user ID being changed is the one logged in, and the only one modified. 
+    // console.log(values); This line is to test that the user ID being changed is the one logged in, and the only one modified.
```

Comment:
You can just delete this line, generally we don't like to leave print statements or commented code in final versions